tests/unit/pynwb_tests/test_misc.py

```diff
@@ -73,6 +75,16 @@ def test_get_spike_times(self):
         self.assertEqual(ut.get_unit_spike_times(0), [0, 1, 2])
         self.assertEqual(ut.get_unit_spike_times(1), [3, 4, 5])
 
+    def test_round_trip(self):
+        ut = UnitTimes('UnitTimes add_spike_times unit test')
+        ut.add_spike_times(0, [0, 1, 2])
+        ut.add_spike_times(1, [3, 4, 5])
+
+        with ContainerRoundTrip(ut) as ut:
+            pass
+            # self.assertEqual(ut.get_unit_spike_times(0), [0, 1, 2])
+            # self.assertEqual(ut.get_unit_spike_times(1), [3, 4, 5])
```

Comment:
Why are these commented out?

Reply:
to point that the current error is with reading the file, not with the read values agreeing the written values, as one might expect. They should be uncommented now.